addons/l10n_ng/demo/demo_company.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="partner_demo_company_ng" model="res.partner">
+        <field name="name">NG Company</field>
+        <field name="vat">12345678-1234</field>
+        <field name="company_registry">RC123456</field>
+        <field name="street">Adeola Odeku St 1</field>
+        <field name="city">Lagos</field>
+        <field name="country_id" ref="base.ng"/>
+        <field name="zip">106104</field>
+        <field name="phone">+2341234567890</field>
```

Comment:
```suggestion
        <field name="phone">+2342011234567</field>
```
Added [area code](https://en.wikipedia.org/wiki/Telephone_numbers_in_Nigeria#Calling_formats) to Lagos.